packages/framework/ai-collab/src/explicit-strategy/agentEditReducer.ts

```diff
@@ -84,6 +102,9 @@ function getSchemaIdentifier(content: TreeEditValue): string | undefined {
 			if (isFluidHandle(content)) {
 				return SchemaFactory.handle.identifier;
 			}
+			if (Tree.is(content, NodeKind.Map as unknown as ImplicitAllowedTypes)) {
+				throw new UsageError("Map Nodes are not currently supported in this context");
+			}
```

Comment:
This isn't right. `NodeKind` is an enum, so we're casting a number or string value to a type that represents nodes / fields in SharedTree.

I'm sorry, I gave you bad advice when I recommended handling Map nodes here. I was thinking the input data format was something coming from the SharedTree library. I see now that it's a custom domain we've defined. That domain doesn't support Maps in this context, so we don't need to do anything for them.

We should remove this.

```suggestion
```